install.js

```diff
@@ -104,7 +105,7 @@ function exit(code) {
 function findSuitableTempDirectory() {
   var now = Date.now()
   var candidateTmpDirs = [
-    process.env.TMPDIR || process.env.TEMP || process.env.npm_config_tmp,
+    os.tmpdir(),
```

Comment:
ya, this is not a safe change. also NPM should be filling in process.env.npm_config_tmp, so maybe you have a bad npm release?